video/out/vo_gpu_next.c

```diff
@@ -337,22 +341,26 @@ static void update_overlays(struct vo *vo, struct mp_osd_res res, double pts,
             .tex = entry->tex,
             .parts = entry->parts,
             .num_parts = entry->num_parts,
-            .color.primaries = frame->color.primaries,
-            .color.transfer = frame->color.transfer,
+            .color = {
+                .primaries = PL_COLOR_PRIM_BT_709,
+                .transfer = PL_COLOR_TRC_SRGB,
+            },
             .coords = coords,
         };
 
-        // Reject HDR/wide gamut subtitles out of the box, since these are
-        // probably not intended to match the video color space.
-        if (pl_color_primaries_is_wide_gamut(ol->color.primaries))
-            ol->color.primaries = PL_COLOR_PRIM_UNKNOWN;
-        if (pl_color_transfer_is_hdr(ol->color.transfer))
-            ol->color.transfer = PL_COLOR_TRC_UNKNOWN;
-
         switch (item->format) {
         case SUBBITMAP_BGRA:
             ol->mode = PL_OVERLAY_NORMAL;
             ol->repr.alpha = PL_ALPHA_PREMULTIPLIED;
+            // Infer bitmap colorspace from source
+            if (src) {
+                ol->color = get_mpi_csp(vo, src);
+                // Seems like HDR subtitles are targeting SDR white
+                if (pl_color_transfer_is_hdr(ol->color.transfer)) {
+                    ol->color.hdr.min_luma = 0;
+                    ol->color.hdr.max_luma = PL_COLOR_SDR_WHITE;
```

Comment:
Probably best to just to set the entire `ol->color.hdr` struct to clear out any spurious DV/HDR10+ metadata that might be left in there, e.g.
```c
ol->color.hdr = (struct pl_hdr_metadata) {
    .max_luma = PL_COLOR_SDR_WHITE,
};
```

(Mental note also to add `pl_hdr_metadata_sdr` in libplacebo...)

Reply:
Ok.